contrib/mixin/mixin.libsonnet

```diff
@@ -238,6 +238,20 @@
               summary: 'etcd cluster database growing very fast.',
             },
           },
+          {
+            alert: 'etcdBackendHighFragmentationRatio',
```

Comment:
Thanks @Elbehery - lgtm but couple of small suggestions 1) the `summary` and `description` mixes use of database and backend, we may want to stick with using database. Also, how about renaming `etcdBackendHighFragmentationRatio` to `etcdDatabaseHighFragmentationRatio` ?  2) in-lined comment below.

Reply:
@spzala thanks a lot for your review, will fix it right away 👍🏽

Reply:
Thank you @Elbehery